docs/reference/settings.md

```diff
@@ -367,6 +367,178 @@ pydantic = { path = "/path/to/pydantic", editable = true }
 
 ---
 
+### `build-backend`
+
+Settings for the uv build backend (`uv_build`).
+
+!!! note
+
+    The uv build backend is currently in preview and may change in any future release.
+
+Note that those settings only apply when using the `uv_build` backend, other build backends
+(such as hatchling) have their own configuration.
+
+#### [`data`](#build-backend_data) {: #build-backend_data }
```

Comment:
I wouldn't know why you would use them for a pure Python project, but they exist in the spec so they exist on the build backend config.